src/commands/script-debug.json

```
@@ -34,6 +34,12 @@
                         "token": "NO"
                     }
                 ]
+            },
+            {
+                "name": "engine",
+                "type": "string",
+                "optional": true,
+                "since": "9.0.0"
```

It's already too late to make it to 9.0.

```suggestion
                "since": "9.1.0"
```